recipe/meta.yaml

```
@@ -40,7 +40,9 @@ build:
   ignore_run_exports:
     - python
   number: 1
```

If you're modifying a build but not building a new version, you need to bump the build number:

```suggestion
  number: 2
```